### PR TITLE
SublimeHQ.Sublimetext2 version 2.0.2

### PR DESCRIPTION
**Pull request by ayushthesmarty**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you validated your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/15714)

### CODE REVIEW

**Comment by OfficialEsco** [comment]
Please use the `PackageIdentifier: SublimeHQ.SublimeText.2`

File: manifests/s/SublimeHQ/Sublimetext2/2.0.2/SublimeHQ.Sublimetext2.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# Created using wingetcreate
+
+PackageIdentifier: SublimeHQ.Sublimetext2
+PackageVersion: 2.0.2
+PackageLocale: en-US
+Publisher: SublimeHQ
+PackageName: Sublime Text 2
+License: Copyright c SublimeHQ
+ShortDescription: Sublime text 2
```
Comment (ItzLevvie) [inline manifests/s/SublimeHQ/Sublimetext2/2.0.2/SublimeHQ.Sublimetext2.locale.en-US.yaml:9]:
```suggestion
ShortDescription: The sophisticated text editor for code, markup and prose.
```

File: manifests/s/SublimeHQ/Sublimetext2/2.0.2/SublimeHQ.Sublimetext2.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,11 @@
+# Created using wingetcreate
+
+PackageIdentifier: SublimeHQ.Sublimetext2
+PackageVersion: 2.0.2
+PackageLocale: en-US
+Publisher: SublimeHQ
+PackageName: Sublime Text 2
+License: Copyright c SublimeHQ
```
Comment (ItzLevvie) [inline manifests/s/SublimeHQ/Sublimetext2/2.0.2/SublimeHQ.Sublimetext2.locale.en-US.yaml:8]:
```suggestion
License: Copyright (c) 2006-2012 Sublime HQ Pty Ltd.
```

File: manifests/s/SublimeHQ/Sublimetext2/2.0.2/SublimeHQ.Sublimetext2.locale.en-US.yaml [hunk 3]
```diff
@@ -0,0 +1,11 @@
+# Created using wingetcreate
+
+PackageIdentifier: SublimeHQ.Sublimetext2
+PackageVersion: 2.0.2
+PackageLocale: en-US
+Publisher: SublimeHQ
```
Comment (ItzLevvie) [inline manifests/s/SublimeHQ/Sublimetext2/2.0.2/SublimeHQ.Sublimetext2.locale.en-US.yaml:6]:
```suggestion
Publisher: Sublime HQ Pty Ltd
```

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

**Review by ghost — APPROVED** [review]

File: manifests/s/SublimeHQ/Sublimetext2/2.0.2/SublimeHQ.Sublimetext2.installer.yaml [hunk 4]
```diff
@@ -0,0 +1,11 @@
+# Created using wingetcreate
+
+PackageIdentifier: SublimeHQ.Sublimetext2
```
Comment (ItzLevvie) [inline manifests/s/SublimeHQ/Sublimetext2/2.0.2/SublimeHQ.Sublimetext2.installer.yaml:3]:
```suggestion
PackageIdentifier: SublimeHQ.SublimeText.2
```
You'll also have to update the folder structure using GitHub's desktop client.

File: manifests/s/SublimeHQ/Sublimetext2/2.0.2/SublimeHQ.Sublimetext2.locale.en-US.yaml [hunk 5]
```diff
@@ -0,0 +1,11 @@
+# Created using wingetcreate
+
+PackageIdentifier: SublimeHQ.Sublimetext2
```
Comment (ItzLevvie) [inline manifests/s/SublimeHQ/Sublimetext2/2.0.2/SublimeHQ.Sublimetext2.locale.en-US.yaml:3]:
```suggestion
PackageIdentifier: SublimeHQ.SublimeText.2
```
You'll also have to update the folder structure using GitHub's desktop client.

File: manifests/s/SublimeHQ/Sublimetext2/2.0.2/SublimeHQ.Sublimetext2.yaml [hunk 6]
```diff
@@ -0,0 +1,7 @@
+# Created using wingetcreate
+
+PackageIdentifier: SublimeHQ.Sublimetext2
```
Comment (ItzLevvie) [inline manifests/s/SublimeHQ/Sublimetext2/2.0.2/SublimeHQ.Sublimetext2.yaml:3]:
```suggestion
PackageIdentifier: SublimeHQ.SublimeText.2
```
You'll also have to update the folder structure using GitHub's desktop client.

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

**Pull request closed without merging**